modules/end-user-guide/partials/assembly_building-intellij-editor-images.adoc

```diff
@@ -0,0 +1,21 @@
+:parent-context-of-assembly_building-intellij-editor-images: {context}
+
+[id="assembly_building-intellij-editor-images_{context}"]
+= Building IntelliJ editor images
```

Comment:
I found something useful here:
`JetBrains IDE` = community versions of `IntelliJ IDEA` and `PyCharm`
source: https://github.com/che-incubator/jetbrains-editor-images

Reply:
> Even better: IDEs Based on the IntelliJ Platform﻿ ,https://plugins.jetbrains.com/docs/intellij/intellij-platform.html#ides-based-on-the-intellij-platform

I have a *big* question here:
The next two procedures are described for a specific editor image (`"JetBrains IDE"=IntelliJ IDEA + PyCharm`, as above). The current version of these procedures performs them on (*any*?) `IntelliJ-based editor` image. So, what exactly is your plan here - to explain how to install only `"JetBrains IDE"=IntelliJ IDEA + PyCharm` or how to install *any* `IntelliJ-based editor`?

Reply:
Here are these particular `JetBrains IDEs` (not to be confused with `IntelliJ Platform-based IDEs`):

https://www.jetbrains.com/products/#type=ide
  https://www.jetbrains.com/idea/
  https://www.jetbrains.com/pycharm/
  https://www.jetbrains.com/webstorm/

Reply:
Found another doc to feed the reflexion: https://github.com/che-incubator/jetbrains-editor-images/blob/main/doc/Compatible-IDE.md

Reply:
It is possible to build Android Studio, which is not a JetBrains IDE, but and IDE based on the IntelliJ Platform, so the best title should be:

Building an image for an IDE based on the IntelliJ Platform

https://github.com/eclipse/che/issues/19575